src/routes/(modeles-services)/services/[slug]/orienter/merci/+page.svelte

```diff
@@ -0,0 +1,59 @@
+<script lang="ts">
+  import LinkButton from "$lib/components/display/link-button.svelte";
+  import { arrowLeftLineIcon, flashLightIcon } from "$lib/icons";
+  import Layout from "../layout.svelte";
+  import type { PageData } from "./$types";
+  import { orientation } from "../store";
+  import illustration from "$lib/assets/illustrations/illu-favs.svg";
+  import Notice from "$lib/components/display/notice.svelte";
+
+  export let data: PageData;
+</script>
+
+<Layout {data}>
+  <div
+    class="gap-s46 px-s14 mt-s56 flex flex-col-reverse items-center justify-around rounded-md border border-gray-03 py-s32 lg:flex-row"
+  >
+    <div class="max-w-lg basis-1/2 text-center">
+      <div class="mx-auto mb-s12 h-s24 w-s24">
+        {@html flashLightIcon}
+      </div>
+      <h2 class=" font-bold text-gray-text">
+        Votre demande a été transmise à la structure porteuse du service
+      </h2>
+      <p class="legend">
+        Vous allez recevoir une copie de cette demande par e-mail, dans quelques
+        instants.
+        {#if $orientation.beneficiaryContactPreferences?.includes("email")}
+          <br />Le ou la bénéficiaire recevra également une notice
+          d’information.
+        {/if}
+      </p>
+
+      {#if $orientation.beneficiaryContactPreferences?.length === 1 && ($orientation.beneficiaryContactPreferences[0] === "email" || $orientation.beneficiaryContactPreferences[0] === "phone")}
```

Comment:
```suggestion
      {#if $orientation.beneficiaryContactPreferences?.length === 1 && ($orientation.beneficiaryContactPreferences[0] === "autre" || $orientation.beneficiaryContactPreferences[0] === "telephone")}
```

alors, plutot ça, pour être cohérent avec le titre de la maquette. Mais je pense pas que ça suffise: s'iel a juste un tel et un "autre", on n'a peut-etre pas pu lui envoyer d'email. Est-ce qu'on regarde les préférences, ou la présence des champs eux-même?

Mais, en rapport avec le commentaire précedent, si on a envoyé par email alors que l'email n'est pas une préférence, est-ce qu'on demande au conseiller de l'informer quand meme (ça veut peut-etre dire qu'iel ne regarde jamais ses emails) ?

Reply:
même chose, tu as vu avec Andrei, ou au moins noté ces questions qqpart pour en parler à son retour?